app/controllers/audio_files_controller.rb

```diff
@@ -35,6 +39,17 @@ def destroy
   redirect_to audio_files_path
 end
 
+def update
+  @audio_file = AudioFile.find_by_id(params[:id])
+
+  if @audio_file.update(audio_file_params)
```

Comment:
Need to check for `nil` here in case the audio_file gets deleted before submitting the form (e.g. deleting it from a different browser tab).